Healthy/Classes/Modules/SearchReceipes /Cell/SearchCollectionViewCell.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+class SearchCollectionViewCell: UICollectionViewCell {
+
+    // MARK: Properties
+
+    static let reusableIdentifire = "SearchCollectionViewCell"
```

Comment:
```suggestion
```

Not a good practice, please create `/Extensions/UICollectionView+Helpers.swift` and use it instead. 

Do you know why this code will break?!

```swift
import UIKit

extension UICollectionView {
    
    func register<T: UICollectionViewCell>(_: T.Type) {
        let bundle = Bundle(for: T.self)
        let nib = UINib(nibName: T.reuseIdentifier, bundle: bundle)
        register(nib, forCellWithReuseIdentifier: T.reuseIdentifier)
    }
    
    func dequeueReusableCell<T: UICollectionViewCell>(for indexPath: IndexPath) -> T {
        guard let cell = dequeueReusableCell(withReuseIdentifier: T.reuseIdentifier, for: indexPath) as? T else {
            fatalError("Could not dequeue cell with identifier: \(T.reuseIdentifier)")
        }
        
        return cell
    }
}
```

Reply:
because not UICollectionViewCell classes has reuseIdentifier variable so we have to but it it in extension like so:
extension UICollectionViewCell {
    static var reuseIdentifier: String {
        return String(describing: self)
    }
}